CI/bench/multimodal/custom_user_dir/multimodal_frameworks_template.yaml

```diff
@@ -0,0 +1,7 @@
+AutoGluon_stable:
```

Comment:
We might want to call this `AutoGluon_master_medium`
`max_epochs=10` is by default, so we can ignore it.

Reply:
I have kept `max_epochs` and added a comment next to it as well.
Will help if we want to make changes in the future.